cmd/controller/main.go

```diff
@@ -46,8 +46,9 @@ const (
 )
 
 var (
-	masterURL  string
-	kubeconfig string
+	masterURL    string
+	kubeconfig   string
+	resyncPeriod = 10 * time.Hour
```

Comment:
Could we make this a const instead? I don't see its value changed in tests or anything.